src/go/k8s/apis/redpanda/v1alpha1/console_types.go

```diff
@@ -134,6 +134,10 @@ type Server struct {
 // Schema defines configurable fields for Schema Registry
 type Schema struct {
 	Enabled bool `json:"enabled"`
+
+	// Indication on whether to use the schema registry CA as trust when connecting to the schema registry.
+	// If not set, the public CAs will be used.
+	UseSchemaRegistryCA bool `json:"useSchemaRegistryCA,omitempty"`
```

Comment:
should this be a pointer since we are allowing omitempty?

Reply:
It does not need to be pointer since the default value is false. If pointer, we had to check whether it was nil.